src/components/AiNotemaker.tsx

```
@@ -0,0 +1,1026 @@
+													const assignedMember = members.find(
+														(m) => m.user._id === item.assigneeUserId
+													);
+													return (
+														<div
+															className={cn(
+																"group flex gap-3 p-4 bg-white border rounded-2xl transition-all duration-300 cursor-pointer select-none",
+																item.selected
+																	? "border-green-200 shadow-sm hover:shadow-md hover:border-green-300"
+																	: "border-gray-100 opacity-60 hover:opacity-100 hover:border-gray-300"
+															)}
+															key={i}
```

Biome (2.4.15)

[error] Avoid using the index of an array as key property in an element.

(lint/suspicious/noArrayIndexKey)